scss/foundation-styles/buttons.scss

```diff
@@ -83,8 +83,8 @@
   }
 }
 
-// append external link glyph
-.btn-secondary[href*="//"]:not([href*="foundation.mozilla.org"]) {
+// // append external link glyph
+.btn-secondary[href*="//"]:not([href*="test.example.com"]):not([href*="pulse-api.mofostaging.net"]):not([href*="api.mozillapulse.org"]) {
```

Comment:
this feels like we're using the wrong approach to determine whether something is an external link or not.

Reply:
I think this may have been a temporary solution. I'm open to suggestions 😅 cc/ @mmmavis